.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+    # Manual dispatch for testing
+  workflow_dispatch:
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@v2
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+      - name: Publish package
+        run: ./gradlew packageWinX64 packageLinuxX64
+      - name: Release
+        uses: softprops/action-gh-release@v2
```

Comment:
I really don't know the first thing about GH Actions, but since this is on the libgdx organization now, I think I need to get the OK from @Tom-Ski or someone else in the org before doing anything with third-party actions. I don't even know if softprops is third-party...